plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py

```diff
@@ -1,5 +1,7 @@
 from typing import TYPE_CHECKING, List, Optional, Union
 
+import PIL.Image
```

Comment:
I do not think we need this? We just use PIL.Image.Image so line 17 should be enough right?

Reply:
Yes, you are probably right.